garak/attempt.py

```diff
@@ -174,7 +174,7 @@ def __setattr__(self, name: str, value: Any) -> None:
         if name == "prompt":
             if value is None:
                 raise TypeError("'None' prompts are not valid")
-            assert isinstance(value, str)
+            # assert isinstance(value, str)
```

Comment:
Do we want to remove this line entirely instead of commenting out?

Reply:
+1 on removal